L1Trigger/Phase2L1ParticleFlow/plugins/L1NNTauProducer.cc

```diff
@@ -97,10 +97,6 @@ void L1NNTauProducer::produce(edm::Event& iEvent, const edm::EventSetup& iSetup)
     process_SW(*l1PFCandidates, lTaus);
   }
 
-  if (lTaus->empty()) {
```

Comment:
Why do you need to remove this protection?

Reply:
Kindly ping @Duchstf

Reply:
@srimanob Hello, sorry for the late response!

This line turns out to be the cause of many tau being sent to HLT with weird 0 pT:

![image](https://github.com/cms-sw/cmssw/assets/35171417/bfbafb4d-3658-4d48-9d26-1292f67426ff)

We were able to fix it by removing this line.